examples/utils/example-utils/src/containerViewRuntimeFactory.ts

```diff
@@ -49,17 +27,30 @@ export class ContainerViewRuntimeFactory<T> extends BaseContainerRuntimeFactory
 		// and add our default view request handler.
 		super({
 			registryEntries: new Map([[dataStoreFactory.type, Promise.resolve(dataStoreFactory)]]),
-			requestHandlers: [
-				// eslint-disable-next-line import/no-deprecated
-				mountableViewRequestHandler(MountableView, [makeViewRequestHandler(viewCallback)]),
-			],
-			provideEntryPoint: async (containerRuntime: IContainerRuntime) => {
-				const entryPoint =
+			provideEntryPoint: async (
+				containerRuntime: IContainerRuntime,
+			): Promise<IFluidMountableViewEntryPoint> => {
+				const entryPointHandle =
 					await containerRuntime.getAliasedDataStoreEntryPoint(dataStoreId);
-				if (entryPoint === undefined) {
-					throw new Error("default dataStore must exist");
+
+				if (entryPointHandle === undefined) {
+					throw new Error(`Default dataStore [${dataStoreId}] must exist`);
+				}
+
+				const entryPoint = await entryPointHandle.get();
+
+				const view = viewCallback(entryPoint as T);
```

Comment:
Maybe type the handle as `IFluidHandle<T> | undefined` above instead, so we clarify the expected type as early as possible.